Week2/homework/map-filter.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(n => n % 2 !== 0).map(n => n * 2);
```

Comment:
I don't mind using single letters here for the parameter names, as this is sort of an algebraic operation where single letters are customary.